src/move/emojicoin_dot_fun/Move.toml

```diff
@@ -1,5 +1,6 @@
 [addresses]
 emojicoin_dot_fun = "_"
+yellow_heart_market_address = "0x6ba9e8b95825fffbfcc3fc743ec7461c13f334f8b3d5c0a30988b6d4399a003a"
```

Comment:
?

Reply:
Ahh okay I see what is going on here

How about moving this to `dev-addresses` with a comment explaning how this is the resulting address when getting a named object from the `emojicoin_dot_fun` dev-address?

Reply:
Actually, on second though, I think we should remove this and determine it programatically at runtime

Reply:
We could event pass this in using `--named-addresses`?

There's just something about hardcoding this on here, as derived from another `dev-address`, that smells funny